### DISCUSSION TITLE
Docker e openvpn

### DISCUSSION THREAD

**Discussion by vinny74man** [indomus/forum#1516] [HUB: Home Assistant]
Ciao a tutti per delle prove ho appena installato HA core in docker su un raspberry, il raspi è collegato tramite vpn si delle apperecchiature KNX, come faccio a far vedere il KNX anche all'installazione su docker?
Grazie

di solito faccio una installazione in venv e in automatico andava però volevo provare installazione in docker.

**Comment by snakuzzo** [comment]
Non ho capito bene...

Tu hai HA su Docker su un RPI che è collegato come client VPN ad un'altra rete su cui vi sono apparecchiature KNX ?
Se è così ed hai creato il container di HA con network mode "host" dovresti vedere tutt la LAN "remota"

**Comment by KC-inDomus** [comment]
> 

Beh scusa, se lanci il container con network host, dovresti vedere anche la rete vpn sul quale sono attestati i KNX

**Comment by vinny74man** [comment]
scusate ragazzi ma non sono proprio partico di docker come faccio a lanciare il container con netwok host, per l'installazione ho seguito la vostra guida, intendo quella su indomus

**Reply by KC-inDomus** [reply]
vedi [qui](https://docker-docs.netlify.app/network/network-tutorial-host/)

**Comment by vinny74man** [comment]
ok grazie spero di risolvere, altrimenti installo in venv e via.

Grazie

**Reply by KC-inDomus** [reply]
ma no. prova a verificare la tua configurazione, introduci network host se mancante, vedi se la cosa risolve
